app/src/layouts/common/appSidebar/appSidebar.scss

```diff
@@ -155,6 +155,10 @@
     width: 20px;
     height: 20px;
   }
+  i{
```

Comment:
Please add empty spaces between the selector and braces.
Check other places as well.
Looks like we need to add this to our linter.